pkg/operator/starter.go

```diff
@@ -98,10 +98,31 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	); err != nil {
 		return err
 	}
+	if err := resourceSyncController.SyncConfigMap(
+		resourcesynccontroller.ResourceLocation{Namespace: operatorNamespace, Name: "initial-client-ca"},
```

Comment:
Collapsing into a single namespace here means less plumbing for wiring together different informers from different namespaces to trigger.